packages/x-components/src/x-modules/queries-preview/x-module.ts

```diff
@@ -0,0 +1,25 @@
+import { XModule } from '../x-modules.types';
+import { queriesPreviewEmitters } from './store/emitters';
+import { queriesPreviewXStoreModule } from './store/module';
+import { QueriesPreviewXStoreModule } from './store/types';
```

Comment:
Not from this PR, but, can you change the imports in `x-modules/extra-params/x-module.ts` so it doesn't use the './store' barrel?

````javascript
import { extraParamsEmitters } from './store/emitters';
import { extraParamsXStoreModule } from './store/module';
import { ExtraParamsXStoreModule } from './store/types';
````

Reply:
Sure!